chainer/functions/array/spatial_transformer_sampler.py

```diff
@@ -131,16 +132,17 @@ def backward_gpu(self, inputs, grad_outputs):
         grid_t = cuda.cupy.ascontiguousarray(grid_t)
         x = cuda.cupy.ascontiguousarray(x)
         gy = cuda.cupy.ascontiguousarray(gy)
-        gx = cuda.cupy.empty_like(x)
-        ggrid_t = cuda.cupy.empty_like(grid_t)
+        gx = cuda.cupy.zeros_like(x)
+        ggrid_t = cuda.cupy.zeros_like(grid_t)
```

Comment:
Could you explain the change?

Reply:
I just left temporary change to find a reason of corrupting data. I will discard the change shortly.

Reply:
Thanks.